controller.go

```diff
@@ -132,32 +132,7 @@ func NewController(
 
 			if ok {
```

Comment:
now that you are changing this, maybe you can also change this
```
if !ok {
    return
}

if len(r.Status.HostID) == 0 {
    return
}
```

from https://github.com/inlets/inlets-operator/pull/75

Reply:
Updated per suggested.